with-graphql/codegen.ts

```diff
@@ -0,0 +1,12 @@
+import type { CodegenConfig } from "@graphql-codegen/cli";
+
+const config: CodegenConfig = {
```

Comment:
I think there was a simpler split we can recommend here, in terms of best practices that avoids the API call. Basically, it's possible to write the schema file while running (or separate out the TS files for the schema and import them)

I can have a look later how Yoga recommends doing that (basically Pothos has this built-in as far as I remember but Yoga might also support this in some way)